models.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose'
+
+
+export const ArtistDetail = mongoose.model('ArtisDetail', {
+ id: Number,
```

Comment:
Mongoose adds unique _id field so it is no need to add one by hand. This will make your work harder when you will start adding new records to the database later.